README.md

```diff
@@ -20,58 +20,42 @@ Learn about our [design principles](/docs/developer/design-principles.md) and [a
+the *edge* version built from the [latest commit](https://github.com/nginxinc/nginx-gateway-fabric/commits/main)
 from the main branch.
 
-To use NGINX Kubernetes Gateway, you need to have access to:
-
-- An NGINX Kubernetes Gateway image.
-- Installation manifests.
-- Documentation and examples.
-
-It is important that the versions of those things above match.
-
-The table below summarizes the options regarding the images, manifests, documentation and examples and gives your links
-to the correct versions:
-
-| Version        | Description                              | Image                                                                                                                                                                                                                                                                                  | Installation Manifests                                                                | Documentation and Examples                                                                                                                                                     |
-|----------------|------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| Latest release | For experimental use                     | Use the 0.6.0 image from [GitHub](https://github.com/nginxinc/nginx-kubernetes-gateway/pkgs/container/nginx-kubernetes-gateway)                                                                                                                                                        | [Manifests](https://github.com/nginxinc/nginx-kubernetes-gateway/tree/v0.6.0/deploy). | [Documentation](https://github.com/nginxinc/nginx-kubernetes-gateway/tree/v0.6.0/docs). [Examples](https://github.com/nginxinc/nginx-kubernetes-gateway/tree/v0.6.0/examples). |
-| Edge           | For experimental use and latest features | Use the edge image                                                                                                                                                         from [GitHub](https://github.com/nginxinc/nginx-kubernetes-gateway/pkgs/container/nginx-kubernetes-gateway) | [Manifests](https://github.com/nginxinc/nginx-kubernetes-gateway/tree/main/deploy).   | [Documentation](https://github.com/nginxinc/nginx-kubernetes-gateway/tree/main/docs). [Examples](https://github.com/nginxinc/nginx-kubernetes-gateway/tree/main/examples).     |
-
```

Comment:
Removed this section since it doesn't offer much value. Our installation guide already covers this.

Reply:
this section clearly gives the link to the correct version of the docs and examples.
Otherwise, it is hard to find the correct versioned link.
I think we should retain it.

Reply:
Ok, but it needs to be reworked a bit.

Reply:
I'll just revert to the table.